sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -279,4 +282,57 @@ export class EducationProgramOfferingControllerService {
       ),
     );
   }
+
+  /**
+   * Transformation util for Program Offering.
+   * @param offering
+   * @param hasExistingApplication is the offering linked to any application.
+   * @returns program offering.
+   */
+  async transformToProgramOfferingDTO(
+    offering: EducationProgramOffering,
+    hasExistingApplication?: boolean,
+  ): Promise<EducationProgramOfferingAPIOutDTO> {
+    const validatedOffering = await this.offeringService.validateOfferingById(
+      offering.id,
+    );
+
+    return {
+      id: offering.id,
+      offeringName: offering.name,
+      studyStartDate: getISODateOnlyString(offering.studyStartDate),
```

Comment:
Why are we using getISODateOnlyString here?

Reply:
This method was just moved from the DTO to here and I did not realize it. It is probably a leftover from a previous task. I will remove it.